Project/Sources/Classes/OAuth2Provider.4dm

```diff
@@ -631,6 +666,10 @@ Function _useJWTBearer() : Boolean
 	
 	return (This.grantType="urn:ietf:params:oauth:grant-type:jwt-bearer")
 	
+Function _useJWTBearerAssertionType() : Boolean
+	
+	return (OB Is defined(This; "_thumbprint") & (This._thumbprint#""))
```

Comment:
`&&` not `&`
to not evaluate right part

another way (that could failed if passing other type of objet in _thumbprint)
```4d
return Length(String(This._thumbprint))>0
```